lib/src/rules/type_init_formals.dart

```diff
@@ -60,17 +62,32 @@ class _Visitor extends SimpleAstVisitor<void> {
   @override
   void visitFieldFormalParameter(FieldFormalParameter node) {
     var nodeType = node.type;
-    if (nodeType != null) {
-      var cls = node.thisOrAncestorOfType<ClassDeclaration>()?.declaredElement;
-      if (cls != null) {
-        var field = cls.getField(node.identifier.name);
-        // If no such field exists, the code is invalid; do not report lint.
-        if (field != null) {
-          if (nodeType.type == field.type) {
-            rule.reportLint(nodeType);
-          }
-        }
+    if (nodeType == null) return;
+    var paramElement = node.declaredElement;
+    if (paramElement is! FieldFormalParameterElement) return;
+
+    var field = paramElement.field;
```

Comment:
Formatting of code in this method is inconsistent with the next method.

Reply:
Good catch.  Fixing.  Thanks!